src/prng/os-prng.lisp

```diff
@@ -16,7 +16,22 @@
     seq)
   ;; FIXME: this is _untested_!
   #+(and win32 sb-dynamic-core)(sb-win32:crypt-gen-random num-bytes)
-  #-(or unix (and win32 sb-dynamic-core))(error "Your platform does not have a supported random source."))
```

Comment:
The repetition between this & `OS-RANDOM-SEED` is pretty ugly at this point, I'll admit.